packages/frontend/editor-ui/src/stores/settings.store.ts

```diff
@@ -64,7 +64,7 @@ export const useSettingsStore = defineStore(STORES.SETTINGS, () => {
 		secureCookie: settings.value.authCookie.secure,
 	}));
 
-	const isEnterpriseFeatureEnabled = computed(() => settings.value.enterprise);
+	const isEnterpriseFeatureEnabled = computed(() => settings.value.enterprise ?? {});
```

Comment:
For convenience in setting up mock in tests, tolerate the `enterprise` property to be missing